lib/cucumber/formatter/html_builder.rb

```diff
@@ -51,6 +49,18 @@ def format_features!(features)
 
       private
 
+      def string_to_embed(type: nil, src: nil, label: nil, id: nil)
+        raise ::ArgumentError, 'missing required argument' unless type && src && label && id # for Ruby 2.0 compatibility
+        raise InvalidEmbedTypeError unless VALID_EMBED_TYPES.include?(type)
+
+        if type == :image
```

Comment:
I'm being picky, but you could swap the `if` around here and use `:text:` instead. That would mean the first case is a one-liner, and you could use `return` on that one line, meaning you can remove the `else`.